js/MineSweeper.js

```diff
@@ -478,6 +516,7 @@ jQuery(function ($) {
 
             $('#mine_flag_display').val(numMines);
             $('#timer').val(0);
+            $('#pause').hide();
```

Comment:
please make pause referenced by css class, not id. Ideally we want no id elements referenced anywhere in the code base. The reason for this is because you can only have 1 element of the same ID in the DOM at the same time.